src/container/Toast.container.tsx

```diff
@@ -20,7 +20,7 @@ export const ToastContainer = observer(() => {
     <View
       style={[
         tw(`absolute p-4 rounded`),
-        {bottom: 20, left: 20, right: 20},
+        {...(latestToast.position === 'top' ? {top: 20} : {bottom: 20}), left: 20, right: 20},
```

Comment:
@ospfranco added the position option to prevent the toast from hiding the last input and save button on `AddToken` component. Other toasts are still displayed on bottom

Reply:
coolio!